tests/failpoints/cases/test_pd_client.rs

```diff
@@ -84,14 +84,14 @@ fn test_pd_client_deadlock() {
             func();
             tx.send(()).unwrap();
         });
+        // Only allow to reconnect once for a func.
```

Comment:
What's it for?

Reply:
The reconnection interval is 100ms. It's possible the test becomes fragile due to frequent reconnection. I ban it after the first reconnection.